crates/csharp/src/csproj.rs

```diff
@@ -190,9 +195,17 @@ impl CSProjectMonoBuilder {
         </PropertyGroup>
 
         <ItemGroup>
-          <NativeFileReference Include=\"{camel}_component_type.o\" Condition=\"Exists('{camel}_component_type.o')\"/>
+          <_WasiLinkStepArgs Include=\"-Wl,--component-type,{camel}_component_type.wit\" />
+          <!-- both versions of these seem to fail to find the .o
```

Comment:
Wit-bindgen doesn't produce the object file directly anymore,

I've used:
```
<CustomLinkerArg Include="@(WitComponentImports->Replace('\', '/')->'-Wl,--component-type,&quot;%(Identity)&quot;')" />
```

In https://github.com/bytecodealliance/componentize-dotnet/pull/35/files#diff-7f4dfa8be7454aa047e35760bfbb0f95c0e1402fd34943606ccd9975e45b8b71 

Not sure if `_WasiLinkStepArgs` is the same?

Reply:
it's another case of bad coevolution https://github.com/dotnet/runtime/pull/107194 but they will be